lms/djangoapps/grades/new/course_grade.py

```diff
@@ -52,6 +52,19 @@ def locations_to_weighted_scores(self):
                 locations_to_weighted_scores.update(subsection_grade.locations_to_weighted_scores)
         return locations_to_weighted_scores
 
+    @lazy
+    def grade_summary(self):
```

Comment:
To avoid confusion between this `grade_summary` method and the pre-existing `summary` method, can you rename this to `grade_value`?  Or is there a better name?